components/camera/transformpipeline/depth_edges.go

```diff
@@ -15,9 +15,9 @@ import (
 )
 
 type depthEdgesAttrs struct {
-	HiThresh   float64 `json:"high_threshold"`
-	LoThresh   float64 `json:"low_threshold"`
-	BlurRadius float64 `json:"blur_radius"`
+	HiThresh   float64 `json:"high_threshold"` // TODO(RSDK-568): units?
+	LoThresh   float64 `json:"low_threshold"`  // TODO(RSDK-568): units?
+	BlurRadius float64 `json:"blur_radius"`    // TODO(RSDK-568): units?
```

Comment:
Hi Threshold and Lo threshold are numbers between 0 and 1, so I guess they are percentages.
blur radius has units of pixels.